src/App.css

```diff
@@ -32,6 +32,7 @@
   from {
     transform: rotate(0deg);
   }
+
```

Comment:
- [ ] Most of the styles here are unnecessary. Please remove them or the App.css file itself.